templates/agents.html

```diff
@@ -76,8 +76,9 @@ <h3 id="modal-paw"></h3>
                 <hr>
                 <table class="obfuscation-table" border=1 frame=void rules=rows>
                     <tr>
-                        <td>Contact</td>
-                        <td><p id="modal-contact"></p></td>
+                        <!--TODO:  Dynamically list available contact options-->
```

Comment:
1) let's remove the TODO comment and go ahead and put in the loop through contacts right now. we should be able to loop through each contact on an agent to display (the data should be on the page in the jinja2 variable).
2) the select/drop-down box is about 2X taller than the other rows in this table. can you make it shorter to match the row size?